internal/core/entities/scheduler_test.go

```diff
@@ -115,3 +115,47 @@ func TestNewScheduler(t *testing.T) {
 		require.NotNil(t, err)
 	})
 }
+
+func TestIsMajorVersion(t *testing.T) {
+	tests := map[string]struct {
+		currentScheduler *entities.Scheduler
+		newScheduler     *entities.Scheduler
+		expected         bool
+	}{
+		"port range should be a major update": {
```

Comment:
Hmmmm... using table-driven tests here, right? I like this approach and I think we should adopt it in more tests, what do you guys think? @lyracampos @caiordjesus ref: https://dev.to/boncheff/table-driven-unit-tests-in-go-407b, https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
This tests already exists before. I just moved it to new file. I didn't know Table Driven but I found it very interesting. for me we adopted

Reply:
Very nice!!!